components/chainhook-cli/Cargo.toml

```diff
@@ -51,7 +53,6 @@ features = ["lz4", "snappy"]
 [dev-dependencies]
 criterion = "0.3"
 redis = "0.21.5"
-clarity-repl = "=1.7.1"
```

Comment:
Apparently not need anymore 🤔

Reply:
ah yes, we're now re-exporting the repl in `stacks-rpc-client`, and using it from there to simplify dependencies.